src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DocumentMapping/AbstractRazorDocumentMappingService.cs

```diff
@@ -375,18 +344,13 @@ private static bool TryMapToGeneratedDocumentPositionInternal(IRazorGeneratedDoc
 
         static LinePosition GetGeneratedPosition(IRazorGeneratedDocument generatedDocument, int generatedIndex)
         {
-            var generatedSource = GetGeneratedSourceText(generatedDocument);
-            return generatedSource.GetLinePosition(generatedIndex);
+            var generatedSource = generatedDocument.GetGeneratedSourceText();
+            return generatedSource.Lines.GetLinePosition(generatedIndex);
```

Comment:
I think you can elide `Lines` here. We have an extension method on `SourceText`.

Reply:
Whoops, thanks. Looks like an accidental revert from a merge